### PR TITLE
feat(GameBanana): Add presence

### PR DESCRIPTION
**Pull request by MegAmi24**
**Describe the changes in this pull request:**
Adds a presence for `GameBanana`

<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

image
image
image
image
image
image
image
</br>
</details>

### CODE REVIEW

**Comment by MegAmi24** [comment]
Any idea what Lint wants me to fix? Doesn't seem to give any specifics.

**Comment by theusaf** [comment]
> Any idea what Lint wants me to fix? Doesn't seem to give any specifics.

> `[warn] Code style issues found in the above file(s). Forgot to run Prettier?`

Make sure you run the linters:
```sh
npx eslint websites/G/GameBanana/presence.ts --fix
npx prettier --write websites/G/GameBanana/presence.ts
```

**Comment by theusaf** [comment]
You'll also want to indent `metadata.json` and `tsconfig.json` using tabs.

File: websites/G/GameBanana/presence.ts [hunk 1]
```diff
@@ -0,0 +1,383 @@
+const presence = new Presence({ clientId: "958520351158050887" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let header: HTMLElement;
```
Comment (EncryptedDev) [inline websites/G/GameBanana/presence.ts:4]: The value of `header` is never changed, you should use a constant instead. 
The type `HTMLElement` should not be used in this case as you can assign the proper type of the element instead

**Comment by EncryptedDev** [comment]
> > Any idea what Lint wants me to fix? Doesn't seem to give any specifics.
> 
> > `[warn] Code style issues found in the above file(s). Forgot to run Prettier?`
> 
> Make sure you run the linters:
> 
> ```shell
> npx eslint websites/G/GameBanana/presence.ts --fix
> npx prettier --write websites/G/GameBanana/presence.ts
> ```

You should not format like this. We have scripts for this for a reason `yarn lint` (or `yarn format`)

File: websites/G/GameBanana/presence.ts [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "gblogo",
+			startTimestamp: browsingTimestamp
```
Comment (EncryptedDev) [inline websites/G/GameBanana/presence.ts:9]: Not properly formatted, run `yarn lint`

File: websites/G/GameBanana/presence.ts [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+	// Check if we're on the site
+	if (document.location.hostname === "gamebanana.com") {
+		header = document.querySelector("#PageTitle");
+
+		// Homepage
+		if (!header) presenceData.details = "Viewing the Homepage";
+		// 404
+		else if (header.textContent.includes("Tango Down!"))
+			presenceData.details = "Tango Down!";
+		// Discussions
```
Comment (EncryptedDev) [inline websites/G/GameBanana/presence.ts:76]: You do not need comments when you can infer the page from the pathname on the next line

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/G/GameBanana/presence.ts (hunk 2 above)
Comment (MegAmi24) [reply websites/G/GameBanana/presence.ts:9]: Running that didn't change the formatting.

Comment (EncryptedDev) [reply websites/G/GameBanana/presence.ts:9]: Worked fine for me

Comment (MegAmi24) [reply websites/G/GameBanana/presence.ts:9]: I've ran `yarn lint` multiple times, both with and without `--fix`. The command doesn't change the file at all. Running `yarn lint:ci` tells me "All matched files use Prettier code style!", so I have no idea what I'm doing wrong.

**Review by EncryptedDev — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**